parsl/channels/oauth_ssh/oauth_ssh.py

```diff
@@ -0,0 +1,140 @@
+import getpass
+import logging
+import select
+import paramiko
+import socket
+
+from parsl.errors import OptionalModuleMissing
+from parsl.channels.ssh.ssh import SSHChannel
+
+try:
+    from oauth_ssh.ssh_service import SSHService
+    from oauth_ssh.oauth_ssh_token import find_access_token
+    _oauth_ssh_enabled = True
+except (ImportError, NameError, FileNotFoundError):
+    _oauth_ssh_enabled = False
+
+
+logger = logging.getLogger(__name__)
+
+
+class OAuthSSHChannel(SSHChannel):
+    """SSH persistent channel. This enables remote execution on sites
+    accessible via ssh. This channel uses Globus based OAuth tokens for authentication.
+    """
+
+    def __init__(self, hostname, username=None, script_dir=None, envs=None, port=22, **kwargs):
```

Comment:
is this explicit **kwargs necessary? I don't see it used in either `OAuthSSHChannel` or `SSHChannel`